addons/l10n_ma/views/res_partner_views.xml

```diff
@@ -5,13 +5,9 @@
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="account.view_partner_property_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='company_registry']" position="attributes">
-                <attribute name="nolabel">1</attribute>
+            <xpath expr="//field[@name='vat']" position="after">
+                <field name="company_registry" string="ICE" invisible="country_code != 'MA'"></field>
```

Comment:
Why not use the new field `company_registry_label`, in `base` directly?

Reply:
From my understanding, we can't use a field to define the label here in the view (it's done before for `vat_label` through the `_get_view(..)` method but the vat label depends on the company installed) but in our case here it depends on the country code selected through the form view so we need it to be dynamic and displayed when the user chooses MA as a country.

Reply:
Can we just use something like this in `base`?
```xml
<label for="company_registry"><field name="company_registry_label"/></label>
<field name="company_registry" nolabel="1"/>
```

The value of the computed field should be correct, no?

Reply:
Unfortunately, we can't do that as the `<label>` here ignores the `<field>` inside it and only accepts the attribute `string`, and set it as the label text
https://github.com/odoo/odoo/blob/9e9f01504a0b2754ad62fe435bca1a86653c4812/addons/web/static/src/views/form/form_compiler.js#L491-L515